profile/profile.go

```diff
@@ -0,0 +1,31 @@
+package profile
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/micromdm/micromdm/profile/internal/profileproto"
+)
+
+type Mobileconfig []byte
+
+type Profile struct {
+	Identifier string
```

Comment:
On second (third? fourth?) thought — I think it also makes sense to leave them in both the protobuf and the Profile type. it's the primary identification method of a unique profile — both by devices and micromdm. that is its used often enough to keep the identifier around separately even if it is duplicated in the profile data itself.

However, that brings up the problem of synchronization. I think perhaps the best bet, at least for now, would be to double-check them before persisting or loading for the first time - try to make sure the profile type matches the identifier. this would save parsing the mobileconfig all the time, for example.